algorithms/C/searching/linearsearch.c

```diff
@@ -0,0 +1,29 @@
+/* Program to search data in given array using linear search */
+#include<stdio.h>
+int main()
+{
+    int len;
+    printf("Enter the lenght of array");
+    scanf("%d",&len); //scanning the legth of array
+}
+int arr[len];  //created an array of size 'len'.
+printf("Enter the element of array:\n");
+for(int i=0;i<len;i++)
+{
+    scanf("%d",&arr[i]);  //storing all elments in array
+}
+int search;
```

Comment:
it would be better a looks good if you declare all data type in the starting of the function